landoscript/docker.d/worker.yml

```diff
@@ -3,3 +3,14 @@ artifact_dir: { "$eval": "ARTIFACTS_DIR" }
 verbose: { "$eval": "VERBOSE == 'true'" }
 lando_api: { "$eval": "LANDO_API" }
 lando_token: { "$eval": "LANDO_TOKEN" }
+# treestatus prod is used for dev and prod landoscript because all landoscript
+# deployments want real tree status information for whichever repos they make
+# changes to. (in dev, these are non-production repositories.)
+treestatus_url: https://treestatus.prod.lando.prod.cloudops.mozgcp.net
+github_config:
+  app_id:
+    # note: prod covers both L3 and L1 workers (the latter is sometimes called "fake-prod", but not here!)
+    $if: 'env == "prod"'
+    then: prod_app_id # releng-readonly
+    else: dev_app_id # releng-readonly-dev
+  privkey_file: { "$eval": "GITHUB_PRIVATE_KEY_FILE" }
```

Comment:
Would it be useful to have a config schema?

Reply:
Indeed! I will add that to my list of follow-ups.